core/shared/src/main/scala/scalaz/zio/ZIO.scala

```diff
@@ -1078,219 +1078,219 @@ sealed trait ZIO[-R, +E, +A] extends Serializable { self =>
   def tag: Int
 }
 
-private[zio] trait ZIOFunctions extends Serializable {
-  // ALL error types in this trait must be a subtype of `UpperE`.
-  type UpperE
-  // ALL environment types in this trait must be a supertype of `LowerR`.
-  type LowerR
+object ZIO {
```

Comment:
`object ZIO extends Serializable`

Reply:
This is a huge cleanup, by the way! It's going to make the Scaladoc look much better and improve type inference / error messages everywhere.